doc/source/user/absolute_beginners.rst

```diff
@@ -586,8 +587,7 @@ the most rapidly. The matrix is stored by rows, making it a **Row-major
 language**. What you do for C or Fortran depends on whether it's more important
 to preserve the indexing convention or not reorder the data.
 
-`Learn more about shape manipulation here
-<https://numpy.org/devdocs/user/quickstart.html#shape-manipulation>`_.
+`Learn more about shape manipulation here <https://numpy.org/devdocs/user/quickstart.html#shape-manipulation>`_.
```

Comment:
This should be a relative link, not an absolute one. Same throughout. @mattip might remember how to make that work, I'd need to look it up.

Reply:
Yes, this needs fixing. See https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role for details. So in this case, the quickstart guide has a label:
```
.. _quickstart.shape-manipulation:
```
and it can be referenced here with
```
:ref:`_quickstart.shape-manipulation`
```
That will give it the section title as text of the hyperlink. If you want to change the text, use:
```
:ref:` some text <_quickstart.shape-manipulation>`
```